Equinox/GameObject.cpp

```diff
@@ -0,0 +1,73 @@
+﻿#include "GameObject.h"
+#include "BaseComponent.h"
+#include "Globals.h"
+#include <GL/glew.h>
+
+GameObject::GameObject()
+{
+}
+
+GameObject::~GameObject()
+{
+}
+
+void GameObject::SetParent(GameObject* new_parent)
+{
+	_parent = new_parent;
```

Comment:
Add this object to the children list of the parent and remove it from the child list of the previous one